ts/Dashboards/EditMode/Sidebar.ts

```diff
@@ -372,6 +374,8 @@ class Sidebar {
         }
     };
 
+    public updatedSettings: any = {};
```

Comment:
Can we add something more elegant than `any`?

Reply:
@bre1470 any particular type comes to your mind? I thought of using the `Partial<HighchartsComponents.EditableOptions>` But I am trying to index the object by string, which is not defined. Any Ideas here?

Reply:
How about `Record<string, object>`

Reply:
`Partial` looks good. Midterm should we merge `EditableOptions` into the regular `Options` and instead have a type alias which makes use of `Pick<Options, ('key1'|'key2'|...)>`.

Reply:
In my opinion, this part will be refactored nevertheless, so the type can be delt with later 🤔

Reply:
Add a `@todo`.